src/dvs2tensor.cpp

```diff
@@ -0,0 +1,296 @@
+#define LIBCAER_FRAMECPP_OPENCV_INSTALLED 0
+
+#include <libcaercpp/devices/davis.hpp>
+#include <libcaer/devices/davis.h>
+#include "dvs2tensor.hpp"
+
+#include <cstddef>
+#include <torch/csrc/autograd/python_variable.h>
+#include <torch/extension.h>
+#include <torch/script.h>
+#include <torch/torch.h>
+#include <type_traits>
+
+// ================================
+#include <atomic>
+#include <csignal>
+
+//=================================
+// include guard
+#ifndef PYBIND11_H
+#define PYBIND11_H
+#include <pybind11/pybind11.h>
+#endif
+
+#ifndef STL_H
+#define STL_H
+#include <pybind11/stl.h>
+#endif
+
+namespace py = pybind11;
+
+
+// Constructor 
+DVSDataConv::DVSDataConv(uint32_t interval, uint32_t bfsize){
+    container_interval = interval; 
+    buffer_size = bfsize;
+}
+
+void DVSDataConv::globalShutdownSignalHandler(int signal) {
+    static atomic_bool globalShutdown(false);
+    // Simply set the running flag to false on SIGTERM and SIGINT (CTRL+C) for global shutdown.
+    if (signal == SIGTERM || signal == SIGINT) {
+        globalShutdown.store(true);
+    }
+}
+
+void DVSDataConv::usbShutdownHandler(void *ptr) {
+    static atomic_bool globalShutdown(false);
+	(void) (ptr); // UNUSED.
+
+	globalShutdown.store(true);
+}
+
+// Open a DAVIS, given ID, and don't care about USB bus or SN restrictions.
+void DVSDataConv::connect2camera(int ID){
```

Comment:
function name shouldn't contain a number

Reply:
connectToCamera

Reply:
Does this also hold for classes (e.g. DVS2UDP class)?

Reply:
Generally speaking I think it is bad form. "To" is just one more syllable and in my mind 2 looks unprofessional.

Reply:
Ok, I updated the respective file and class names.